plugin/src/main/java/org/jenkinsci/plugins/casc/ConfigurationAsCode.java

```diff
@@ -160,6 +158,10 @@ public void doReplace(StaplerRequest request, StaplerResponse response) throws E
             sources = Collections.singletonList(newSource);
             //TODO: here should be dry run
             configureWith(getConfigFromSources(getSources()));
+            CasCGlobalConfig config = GlobalConfiguration.all().get(CasCGlobalConfig.class);
```

Comment:
you don't need doReplace anymore as changing this parameter is handled by CasCGlobalConfig#configure